example.py

```diff
@@ -21,8 +21,8 @@
 engine = create_engine(
     "rockset_sqlalchemy://",
```

Comment:
also update this to `rockset://`?

Reply:
Either `rockset://` or `rockset_sqlalchemy://` should work, but yes, I think it's a good idea to encourage people to use `rockset://`. I'll fix that.